proposals/README.md

```diff
@@ -0,0 +1,241 @@
+        /// Wearable config
+        address wearables = addresses.getAddress("ERC1155_MAX_SUPPLY_MINTABLE_WEARABLES");
+        for (uint256 i = 0; i < wearableTokenIDMaxSupplySettings.length; i++) {
+            _pushTimelockAction(
+                wearables,
+                abi.encodeWithSignature(
+                    "setSupplyCap(uint256,uint256)",
+                    wearableTokenIDMaxSupplySettings[i].tokenId,
+                    wearableTokenIDMaxSupplySettings[i].maxSupply
+                ),
+                string(
+                    abi.encodePacked(
+                        "Set wearable tokenId ",
+                        wearableTokenIDMaxSupplySettings[i].tokenId,
+                        " to max supply ",
+                        wearableTokenIDMaxSupplySettings[i].maxSupply
+                    )
+                )
+            );
+        }
+    }
+
+    function _run(Addresses addresses, address) internal override {
+        this.setDebug(true);
+
+        _simulateTimelockActions(
+            addresses.getAddress("ADMIN_TIMELOCK_CONTROLLER"),
+            addresses.getAddress("ADMIN_MULTISIG"),
+            addresses.getAddress("ADMIN_MULTISIG")
+        );
+    }
+
+    function _validate(Addresses addresses, address) internal override {
+        ERC1155MaxSupplyMintable wearable = ERC1155MaxSupplyMintable(
+            addresses.getAddress("ERC1155_MAX_SUPPLY_MINTABLE_WEARABLES")
+        );
+
+        /// Verfiy Wearable
+        for (uint256 i = 0; i < wearableTokenIDMaxSupplySettings.length; i++) {
+            uint256 tokenId = wearableTokenIDMaxSupplySettings[i].tokenId;
+            uint256 maxSupply = wearableTokenIDMaxSupplySettings[i].maxSupply;
+            uint256 currentSupply = wearable.totalSupply(tokenId);
+
+            assertEq(wearable.maxTokenSupply(tokenId), maxSupply, "Invalid maxTokenSupply for tokenId");
+            assertEq(wearable.getMintAmountLeft(tokenId), maxSupply - currentSupply, "Invalid getMintAmountLeft for tokenId");
+        }
+    }
+
+    function _validateOnChain(Addresses, address deployer) internal virtual override {}
+
+    function _validateForTestingOnly(Addresses, address deployer) internal virtual override {}
+}
+```
+
+#### Step 2: Add the zip in TestProposals.sol and test
+
+1. Set ENVIRONMENT as 'mainnet', 'qa', 'devnet', 'localnet' or any other environment against which we want to run the integration tests.
+
+2. Update the address json file as required for the new zip.
+
+3. Import new zip in TestProposals.
+
+4. If zip is already executed on the network, add it in `if` block and now you can run integration tests. If zip is not executed on mainnet, replace zipTest with the new zip contract and run integration tests to test the zip. 'localnet' represents anvil test network and all the zips are simulated when running integration tests against it.
+Run:
+```bash
+forge test --match-contract IntegrationTest
```

Comment:
Use NPM to run these commands. Take a look at `package.json`, as that will need some cleaning up too.